drivers/watchdog/wdt_nrfx.c

```diff
@@ -17,6 +17,7 @@ struct wdt_nrfx_data {
 	wdt_callback_t m_callbacks[NRF_WDT_CHANNEL_NUMBER];
 	uint32_t m_timeout;
 	uint8_t m_allocated_channels;
+	bool enabled;
```

Comment:
This is non-blocking but to make it common across other wdt drivers I suggest using either `timeout_installed` or `timeout_valid`

Reply:
Naming does not appear to be consistent across drivers:
https://github.com/zephyrproject-rtos/zephyr/blob/0f776cf0bfe85caed8281636354eb72454053624/drivers/watchdog/wdt_rpi_pico.c#L28